.github/dependabot.yaml

```
@@ -4,3 +4,7 @@ updates:
     directory: back/go
```

ごめん、ついでにここ直しといて
```suggestion
    directory: backend/go
```
今気づいた